composer.json

```diff
@@ -66,7 +66,11 @@
         "phpstan/phpstan-strict-rules": "^1.5",
         "phpstan/phpstan-symfony": "^1.3",
         "phpstan/phpstan-doctrine": "^1.3",
-        "php-cs-fixer/shim": "^3.50"
+        "php-cs-fixer/shim": "^3.50",
+        "phpunit/phpunit": "^9.5",
+        "symfony/browser-kit": "5.4.*",
+        "symfony/css-selector": "5.4.*",
+        "symfony/phpunit-bridge": "^7.0"
```

Comment:
Can we use 5.4 of the bridge to be inline with other symfony packages?

Reply:
Done in 4381f4fc